pkg/portutil/portutil.go

```diff
@@ -45,7 +47,7 @@ func splitParts(rawport string) (string, string, string) {
 
 // ParseFlagP parse port mapping pair, like "127.0.0.1:3000:8080/tcp",
 // "127.0.0.1:3000-3001:8080-8081/tcp" and "3000:8080" ...
-func ParseFlagP(s string) ([]gocni.PortMapping, error) {
+func ParseFlagP(cmd *cobra.Command, s string) ([]gocni.PortMapping, error) {
```

Comment:
pkg/* has to be decoupled from cobra

Reply:
Thanks for your review! I'll update my pr asap